src/rnaseq-pipeline.nf

```diff
@@ -69,6 +72,7 @@ process regtools {
         """
         samtools index $bam_file
         regtools junctions extract $bam_file -o ${fastq_file.baseName}.bed
+        python $remove_transgene $params.ref_dir/${ref_gene_file.baseName}.bed ${fastq_file.baseName}.bed ${fastq_file.baseName}_clean.bed
```

Comment:
`remove_transgene` produces `${fastq_file.baseName}_clean.bed`, which is what should go into `REGTOOLS_out_1` under the `output` directive.